src/__integration_tests__/authentication.integration.test.ts

```diff
@@ -656,9 +655,12 @@ describe("Logout tests", () => {
     await page.waitForNavigation();
     const userButton = await getUserButton();
     await userButton.click();
-    const logoutButton = await page.waitForXPath('/html/body/div[5]/div[3]/ul/li');
-    console.log("wait for logout button");
-    await logoutButton.click();
+    await page.waitForSelector("#logout-button-icon");
+    await page.evaluate(()=>{
+      setTimeout(()=>{
+        document.getElementById("logout-button-icon").parentNode.click();
+      },100);
+    })
```

Comment:
```suggestion
    const logoutButtonIcon = await page.waitForSelector("#logout-button-icon");
    const logoutButton = await logoutButtonIcon.getProperty('parentNode');
    await logoutButton.click();
```

What do you think about this rather than adding evaluate with setTimeout?

Reply:
Similar implementation is used into "Should invoke auth-service api with x-rpt-id header" test.

Reply:
ok, done